hack/manifest-gen/assets/charts/eck/templates/NOTES.txt

```diff
@@ -0,0 +1,2 @@
+1. Inspect the operator logs by running the following command:
+   kubectl logs -n {{ .Release.Namespace }} $(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "eck.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
```

Comment:
```suggestion
   kubectl logs -n {{ .Release.Namespace }} sts/{{ .Release.Name }}
```

Should work too and does make any assumptions about the shell in use, or am I missing something?